text/0070-x-track-kusamanetwork.md

```diff
@@ -0,0 +1,160 @@
+X account, but if they are affiliated with Parity or W3F this proposed solution could also offload some of the legal ramifications of making (or not making)
+announcements to the public regarding Kusama. While centralized control of the X account would still be present, it could become totally moot if this RFC is implemented
+and the community becomes totally autonomous in the management of Kusama's X posts.
+
+This solution does not cover every single communication front for Kusama, but it does cover one of the largest. It also establishes a precedent for other communication channels
+that could be offloaded to openGov, provided this proof-of-concept is successful.
+
+Finally, this RFC is the epitome of experimentation that Kusama is ideal for. This proposal may spark newfound excitement for Kusama and help us realize Kusama's potential
+for pushing boundaries and trying new unconventional ideas.
+
+## Stakeholders
+
+This idea has not been formalized by any individual (or group of) KSM holder(s). To my knowledge the socialization of this idea is contained
+entirely in [my recent X post here](https://twitter.com/AdamSteeber1/status/1750541362498302230), but it is possible that an idea like this one has been discussed in
+other places. It appears to me that the ecosystem would welcome a change like this which is why I am taking action to formalize the discussion.
+
+## Explanation
+
+The implementation of this idea can be broken down into 3 primary phases:
+
+### Phase 1 - Track configurations
+
+First, we begin with this RFC to ensure all feedback can be discussed and implemented in the proposal. After the Fellowship and the community come to a reasonable
+agreement on the changes necessary to make this happen, the Fellowship can merge changes into Kusama's runtime to include this new track with appropriate track configurations.
+As a starting point, I recommend the following track configurations:
+
+```
+const APP_X_POST: Curve = Curve::make_linear(7, 28, percent(50), percent(100));
+const SUP_X_POST: Curve = Curve::make_reciprocal(?, ?, percent(?), percent(?), percent(?));
+
+// I don't know how to configure the make_reciprocal variables to get what I imagine for support,
+// but I recommend starting at 50% support and sharply decreasing such that 1% is sufficient quarterway
+// through the decision period and hitting 0% at the end of the decision period, or something like that.
+
+	(
+		69,
+		pallet_referenda::TrackInfo {
+			name: "x_post",
+			max_deciding: 50,
+			decision_deposit: 1 * UNIT,
+			prepare_period: 10 * MINUTES,
+			decision_period: 4 * DAYS,
+			confirm_period: 10 * MINUTES,
+			min_enactment_period: 1 * MINUTES,
+			min_approval: APP_X_POST,
+			min_support: SUP_X_POST,
+		},
+	),
+```
+
+I also recommend restricting permissions of this track to only submitting remarks or batches of remarks - that's all we'll need for its purpose. I'm not sure how
```

Comment:
This doesn't work, but the track origin also could not really call anything problematic.